app/src/main/java/dev/beefers/vendetta/manager/network/utils/CommitsPagingSource.kt

```diff
@@ -2,11 +2,13 @@ package dev.beefers.vendetta.manager.network.utils
 
 import androidx.paging.PagingSource
 import androidx.paging.PagingState
+import dev.beefers.vendetta.manager.domain.manager.PreferenceManager
 import dev.beefers.vendetta.manager.domain.repository.RestRepository
 import dev.beefers.vendetta.manager.network.dto.Commit
 
 class CommitsPagingSource(
-    private val repo: RestRepository
+    private val repo: RestRepository,
+    private val prefs: PreferenceManager
```

Comment:
Not sure if injecting preferences is clean, but seemed to be the intended approach.